compiler/rustc_codegen_llvm/src/lib.rs

```diff
@@ -460,6 +462,7 @@ impl ModuleLlvm {
 impl Drop for ModuleLlvm {
     fn drop(&mut self) {
         unsafe {
+            drop(ManuallyDrop::take(&mut self.tm));
```

Comment:
Can't this be done in-place?
```suggestion
            ManuallyDrop::drop(&mut self.tm);
```